docs/deploy/setup-iomesh.md

```diff
@@ -84,12 +84,15 @@ spec:
 ```
 
 #### Mount Type
-
-IOMesh provides 2 mount types now:
+In `mixed-flash` deployment node, IOMesh provides 2 mount types:
 
 - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
```

Comment:
**MUST** ba --> **MUST** be
paritions --> partitions
one for `journal` and another for `cache`. --> one for `journal` and the other for `cache`.